src/agent/src/config.rs

```diff
@@ -240,8 +258,17 @@ impl FromStr for AgentConfig {
         );
         config_override!(agent_config_builder, agent_config, guest_components_procs);
         #[cfg(feature = "guest-pull")]
-        config_override!(agent_config_builder, agent_config, image_registry_auth);
+        {
+            config_override!(agent_config_builder, agent_config, image_registry_auth);
+            config_override!(
+                agent_config_builder,
+                agent_config,
+                enable_signature_verification
```

Comment:
```suggestion
            config_override!(agent_config_builder, agent_config, enable_signature_verification);
            config_override!(
                agent_config_builder,
                agent_config,
                enable_signature_verification
```

Ohhhhhh-My-Goooooooood. TYPT is the thief?